src/interface.jl

```diff
@@ -240,6 +243,80 @@ function forward(cb::Composed, x)
     return (rv=rv, logabsdetjac=logjac)
 end
 
+-###########
+-# Stacked #
+-###########
+"""
+    Stacked(bs)
+    Stacked(bs, ranges)
+    vcat(bs::Bijector...)
+
+A `Bijector` which stacks bijectors together which can then be applied to a vector
+where `bs[i]::Bijector` is applied to `x[ranges[i]]`.
+
+# Examples
+```
+b1 = Logistic(0.0, 1.0)
+b2 = Identity()
+b = vcat(b1, b2)
+b([0.0, 1.0]) == [b1(0.0), 1.0]  # => true
+```
+"""
+struct Stacked{B, N} <: Bijector where N
+    bs::B
+    ranges::NTuple{N, UnitRange{Int}}
+end
+Stacked(bs) = Stacked(bs, NTuple{length(bs), UnitRange{Int}}([i:i for i = 1:length(bs)]))
+Stacked(bs, ranges) = Stacked(bs, NTuple{length(bs), UnitRange{Int}}(ranges))
+
+Base.vcat(bs::Bijector...) = Stacked(bs)
+
+inv(sb::Stacked) = Stacked(inv.(sb.bs), sb.ranges)
+
+# TODO: Is there a better approach to this?
+@generated function _transform(x, rs::NTuple{N, UnitRange{Int}}, bs::Bijector...) where N
+    exprs = []
+    for i = 1:N
+        push!(exprs, :(bs[$i](x[rs[$i]])))
+    end
+
+    return :(vcat($(exprs...)))
+end
+_transform(x, rs::NTuple{1, UnitRange{Int}}, b::Bijector) = b(x)
+
+function (sb::Stacked)(x::AbstractArray{<: Real})
+    y = _transform(x, sb.ranges, sb.bs...)
+
+    # TODO: maybe tell user to check their ranges?
+    @assert size(y) == size(x)
+
+    return y
+end
+(sb::Stacked)(x::AbstractMatrix{<: Real}) = hcat([sb(x[:, i]) for i = 1:size(x, 2)]...)
+function (sb::Stacked)(x::TrackedArray{A, 2}) where {A}
+    return Tracker.collect(hcat([sb(x[:, i]) for i = 1:size(x, 2)]...))
```

Comment:
Can you remind me of what does `Tracker.collect` do?

Reply:
Basically convert `Array{TrackedReal}` into `TrackedArray`